File: pset6/mario/mario.py
from cs50 import get_int

# Check if height is valid number
while(True):
    height = get_int("Height: ")
    if height <= 8 and height >= 1:
        break

# Create pyramid
for i in range(height - 1, -1, -1):
    for j in range(0, i):
        print(" ", end = "")
    for k in range(0, height - i):
        print("#", end = "")
    print("  ", end = "")
    for k in range(0, height - i):
        print("#", end = "")
    print("")

File: pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("height: ");
    }
    while (height>8 || height<1);
    for (int i=height-1; i>=0; i--)
    {
        for (int j=0; j<i; j++)
        {
           printf(" ");
        }
        for (int k=0; k<height-i; k++)
        {
           printf("#");
        }
        printf("  ");
        for (int k=0; k<height-i; k++)
        {
            printf("#");
        }
        printf("\n");
    }
}